lib/renderer/remote.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const { remote } = require('electron')
+
+module.exports = function (usage) {
```

Comment:
This isn't very easy to understand what the function call to `remote` does.

Can we export a named function, e.g.

```js
module.exports.getRemoteForUsage = (usage) => {

}
```

Just so usages of this method are readable

Reply:
done